inc/packages/templates/config.php

```diff
@@ -0,0 +1,222 @@
+					<p><?php esc_html_e( 'The syntax for the provided file should be in a CSV format:', 'altis' ); ?></p>
+					<pre><?php esc_html_e( 'text, token 1 ... token n, reading 1 ... reading n, part-of-speech tag', 'altis' ); ?></pre>
+					<p><?php esc_html_e( 'For example', 'altis' ); ?>:</p>
+					<pre>東京スカイツリー,東京 スカイツリー,トウキョウ スカイツリー,カスタム名詞</pre>
+				</div>
+				<div class="column">
+					<h3>
+						<label for="user-dictionary-file"><?php esc_html_e( 'File upload', 'altis' ); ?></label>
+						<?php if ( $user_dictionary_file_date && $user_dictionary_uploaded_status ) : ?>
+							<span class="es-package-status es-package-status--<?php echo esc_attr( strtolower( $user_dictionary_uploaded_status ) ); ?>">
+								<?php esc_html_e( 'Status', 'altis' ); ?>: <?php echo esc_html( $user_dictionary_uploaded_status ); ?>
+							</span>
+						<?php endif; ?>
+					</h3>
+					<?php if ( $user_dictionary_file_date ) : ?>
+						<?php if ( $user_dictionary_uploaded_error ) : ?>
+							<div class="notice notice-error">
+								<p><?php echo esc_html( $user_dictionary_uploaded_error ); ?></p>
+							</div>
+						<?php endif; ?>
+						<p>
+							<?php
+							echo esc_html( sprintf(
+								// translators: %s is replaced by the file upload date.
+								__( 'Current file uploaded on %s', 'altis' ),
+								gmdate( get_option( 'date_format', 'Y-m-d H:i:s' ), $user_dictionary_file_date )
+							) );
+							?>
+						</p>
+						<input type="submit" class="components-button is-link is-destructive" name="user-dictionary-remove" value="<?php esc_attr_e( 'Remove user dictionary file', 'altis' ); ?>" />
+					<?php endif; ?>
+					<p>
+						<input type="file" accept="text/plain" id="user-dictionary-file" name="user-dictionary-file" />
+					</p>
+					<p>
+						<input type="submit" class="button button-primary" value="<?php esc_attr_e( 'Update user dictionary', 'altis' ); ?>" />
+					</p>
+				</div>
+			</div>
+			<hr />
+		<?php endif; ?>
+		<h2><?php esc_html_e( 'Synonyms' ); ?></h2>
+		<div class="columns">
+			<div class="column">
+				<p><?php esc_html_e( 'Synonyms allow the search index to treat specific words or phrases as equal. This is useful if:', 'altis' ); ?></p>
+				<ul>
+					<li><?php esc_html_e( 'you have a lot of acronyms or other contractions in your content', 'altis' ); ?></li>
+					<li><?php esc_html_e( 'you have terms in the content that do not appear in the dictionary', 'altis' ); ?></li>
+					<li><?php esc_html_e( 'there are common mis-spellings of search terms', 'altis' ); ?></li>
+					<li><?php esc_html_e( 'you have users from different countries who may use different words for the same thing', 'altis' ); ?></li>
+				</ul>
+				<p><?php esc_html_e( 'The syntax for synonyms is as follows:', 'altis' ); ?></p>
+				<pre># <?php esc_html_e( 'Comments are allowed using a hash at the start the line.', 'altis' ); ?></pre>
```

Comment:
```suggestion
				<pre><?php esc_html_e( '# Comments are allowed using a hash at the start the line.', 'altis' ); ?></pre>
```

Reply:
I don't really get this change - the hash shouldn't be translated as it's to demonstrate the comment.

Reply:
In different languages it may need to be in different places in the sentence; e.g. in RTL languages, there may need to be LTR override to ensure it's displayed in the right place.

Reply:
It's a code example. Is code written RTL in RTL languages?

Reply:
No, which is why there may need to be an LTR override as part of the translated text.

In RTL languages, the document's language may be set to RTL. In order to display this interleaved text directly, an [LTR-override character](https://www.fileformat.info/info/unicode/char/200e/index.htm) may be required.

(The examples not necessarily because it's not interleaved, otherwise I'd have suggested making those translatable too.)

Reply:
TIL, cheers 😄